docs/aca/05-aca-dapr-pubsubapi/TasksNotifierController.cs

```diff
@@ -13,49 +12,23 @@ public class TasksNotifierController : ControllerBase
         private readonly IConfiguration _config;
         private readonly ILogger _logger;
         private readonly DaprClient _daprClient;
+
         public TasksNotifierController(IConfiguration config, ILogger<TasksNotifierController> logger, DaprClient daprClient)
         {
             _config = config;
             _logger = logger;
             _daprClient = daprClient;
         }
 
-        [Dapr.Topic("dapr-pubsub-servicebus", "tasksavedtopic")]
+        //[Topic("dapr-pubsub-servicebus", "tasksavedtopic")]  //Dapr Pub Sub Service Bus
```

Comment:
@simonkurtz-MSFT shall we keep the Service Bus componenet the default one (enabled)?

Reply:
Great question, @tjoudeh! I think what's a bit confusing here may be that there are now two files. I modified the original `TasksNotifierController.cs` a good bit because SendGrid and Service Bus are not yet covered at that point of Module 5. Rather, we are focused on the local Redis cache. The second file, `TasksNotifierController-SendGrid.cs` that ultimately replaces this as the module progresses uses the Service Bus by default like so:

```
[Topic("dapr-pubsub-servicebus", "tasksavedtopic")]  //Dapr Pub Sub Service Bus
[HttpPost("tasksaved")]
```